docs/how-to/code/create-slice/task.yaml

```diff
@@ -12,17 +12,16 @@ execute: |
   apt show openssl
   # [docs:apt-show-openssl-end]
 
-  git clone https://github.com/canonical/chisel
+  # Both the local chisel-releases and root path must be in $HOME for the 
+  # chisel snap to be able to access them.
+  pushd $HOME
   git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases
```

Comment:
Spread test is complaining that a non-empty dir "chisel-releases" already exists in $HOME.

Reply:
y trying to fix that

Reply:
fixed in https://github.com/canonical/rockcraft/pull/456/commits/926ff933a1dbff61287c95620612c7f84da97cbc